android/src/main/java/com/reactnativejitsimeet/JitsiMeetModule.java

```diff
@@ -40,11 +40,11 @@ public String getName() {
     return NAME;
   }
 
-  @ReactMethod
-  public void hangUp() {
-    Intent hangUpBroadcastIntent = new Intent("org.jitsi.meet.HANG_UP");
-    LocalBroadcastManager.getInstance(getReactApplicationContext()).sendBroadcast(hangUpBroadcastIntent);
-  }
+//  @ReactMethod
+//  public void hangUp() {
+//    Intent hangUpBroadcastIntent = new Intent("org.jitsi.meet.HANG_UP");
+//    LocalBroadcastManager.getInstance(getReactApplicationContext()).sendBroadcast(hangUpBroadcastIntent);
+//  }
```

Comment:
why are we commenting this code?

Reply:
> might be easier to explain over the phone.

Basically, we have 2 modes and when upgrading the android Jitsi SDK to v6 it breaks the mode where we load the lib within a view (mode 2).  Since this was a bit of an undertaking to resolve the view mode (mode 2) just so I can run the standalone mode (mode1), I temporarily disabled it by commenting out all the references.  I didn't delete them because I have a feeling we are going to need to support it in the future, especially if we are going to support VC minimize functions (concept in the works for mobile).

Good News
- v6 sdk works on android
- v6 sdk works on iOS (both modes)

Bad News
- mode 1 only for android